lib/utils/error-handler.js

```diff
@@ -20,6 +20,19 @@ process.on("exit", function (code) {
   itWorked = false; // ready for next exit
 });
 
+/**
+ * Ensuring that an error is logged with a request ID if it is present.
+ * @param errorToLog - The error to log.
+ */
+function logReqError(errorToLog) {
+  if (errorToLog && errorToLog.requestId) {
+    log.error(errorToLog);
+    log.error(i18n._("Request ID: ") + errorToLog.requestId);
```

Comment:
Maybe use a placeholder instead of concatenation like in https://github.com/feedhenry/fh-fhc/pull/242/files#diff-cafaed9262e78ddbfa9fe5aab3212a82R46